src/styles.scss

```diff
@@ -21,6 +21,8 @@ body {
     position: sticky;
     top: 0;
     z-index: 1000;
+    box-shadow: 0px 2px 4px -1px rgba(0, 0, 0, 0.2), 0px 4px 5px 0px rgba(0, 0, 0, 0.14),
+        0px 1px 10px 0px rgba(0, 0, 0, 0.12);
```

Comment:
https://material.angular.io/guide/elevation

Reply:
I'm going to add class `mat-elevation-z8` to each individual toolbar.

Reply:
`z8` is way too much elevation.   I think `z2` works.

Reply:
Collapsible appbar already has z2 elevation on it, I think this is the correct size.